### ISSUE TITLE
Executing the backfill script on TimescaleDB 2.11.1 version is broken

### ISSUE THREAD

**Issue by ebreijo** [timescale/timescaledb-extras#35]
Current database Setup:
-  Postgres 13.11
- TimescaleDB 2.11.1

Issue:
Executing the backfill script on TimescaleDB 2.11.1 version is broken with the error below

```
ERROR:  function _timescaledb_internal.time_literal_sql(bigint, regtype) does not exist
LINE 1: SELECT COALESCE(r_end, _timescaledb_internal.time_literal_sq...
                               ^
HINT:  No function matches the given name and argument types. You might need to add explicit type casts.
QUERY:  SELECT COALESCE(r_end, _timescaledb_internal.time_literal_sql(min_time_internal, dimension_row.column_type))
CONTEXT:  PL/pgSQL function decompress_backfill(regclass,regclass,text,boolean,interval,text[],boolean) line 134 at assignment
```

This is working on TimescaleDB 2.9.0 version with Postgres 13.11

**Comment by ebreijo** [comment]
It looks like this issue was caused due to this: https://github.com/timescale/timescaledb/pull/5535

Is there any workaround?

**Comment by ebreijo** [comment]
This dependency issue was addressed in this PR #32
